README.md

```diff
@@ -3,8 +3,93 @@
 Keto is under development and should not be used or treated as working
 software. Everything will probably change.
 
-### Building
+## Pre-requisites
 
+### Install golang
+
+Keto is built against the latest golang which can be installed from https://golang.org/dl/.
+
+### Building keto
+
+Get the source code:
 ```
 $ go get -u github.com/UKHomeOffice/keto/cmd/keto
 ```
+
+Get the correct dependency versions:
+```
+cd ${GOPATH}/src/github.com/UKHomeOffice/keto
+curl -s https://glide.sh/get | sh
+glide install
+```
+
+Build the binary:
+```
+go build -v github.com/UKHomeOffice/keto/cmd/keto
+```
+
+## Usage
+
+### Help
+```
+keto --help
+```
+
+### Create Cluster
+
+You will need to [create](#create_expected_ca_files) or obtain suitable CA certs before running keto.
+Minimal command to create a Kubernetes cluster in AWS:
+```
+keto create cluster testcluster --ssh-key my-aws-key-name --networks subnet-b1d0f8fb --machine-type t2.medium --cloud aws
+```
```

Comment:
does this assume you have a subnet already created? or is that just a random name?